harness/determined/cli/_util.py

```diff
@@ -85,28 +84,27 @@ def make_pagination_args(
 default_pagination_args = make_pagination_args()
 
 
-def login_sdk_client(func: Callable[[argparse.Namespace], Any]) -> Callable[..., Any]:
-    @functools.wraps(func)
-    def f(namespace: argparse.Namespace) -> Any:
-        client.login(master=namespace.master, user=namespace.user)
-        return func(namespace)
-
-    return f
+def unauth_session(args: argparse.Namespace) -> api.UnauthSession:
+    master_url = args.master or util.get_default_master_address()
+    return api.UnauthSession(master=master_url, cert=cli.cert)
 
 
 def setup_session(args: argparse.Namespace) -> api.Session:
     master_url = args.master or util.get_default_master_address()
-    cert = certs.default_load(master_url)
-    retry = api.default_retry()
-
-    return api.Session(master_url, args.user, authentication.cli_auth, cert, retry)
+    utp = authentication.login_with_cache(
+        master_address=master_url,
+        requested_user=args.user,
+        password=None,
+        cert=cli.cert,
+    )
+    return api.Session(master_url, utp, cli.cert, api.default_retry())
 
 
 def require_feature_flag(feature_flag: str, error_message: str) -> Callable[..., Any]:
     def decorator(function: Callable[..., Any]) -> Callable[..., Any]:
         def wrapper(args: argparse.Namespace) -> None:
-            resp = bindings.get_GetMaster(setup_session(args))
-            if not resp.to_json().get("rbacEnabled"):
+            resp = bindings.get_GetMaster(unauth_session(args))
+            if not resp.rbacEnabled:
```

Comment:
I can't tell if you wanted `feature_flag` to be a parameter or if you wanted it `rbacEnabled` only.

Reply:
pre-existing bug